platform-service/src/unstract/platform_service/main.py

```diff
@@ -420,4 +420,4 @@ def handle_custom_exception(error: Any) -> tuple[Response, Any]:
 
 if __name__ == "__main__":
     # Start the server
-    app.run()
+    app.run(host="0.0.0.0", port="3001")
```

Comment:
@hari-kuriakose wouldn't this always start the service at 3001? Will users still be able to change the port with `flask run --port <port_number>`?

Reply:
@chandrasekharan-zipstack The default port and the one mentioned in the docker compose did not match, hence set explicitly.
And yes, this can be overridden via CLI.